app/components/metrics_summary_component/view.html.erb

```diff
@@ -1,11 +1,17 @@
 <div class="app-metrics">
-  <h2 class="govuk-heading-l"><%= t("metrics_summary.heading", formatted_date_range:).html_safe %></h2>
+  <h2 class="govuk-heading-l"><%= t("metrics_summary.heading", number_of_days:, formatted_date_range:).html_safe %></h2>
   <% if error_message.present? %>
     <div class="govuk-inset-text">
         <%= error_message.html_safe %>
     </div>
   <% else %>
-    <p><%= t("metrics_summary.description") %></p>
+    <p>
+      <% if complete_week? %>
+        <%= t("metrics_summary.description.complete_week") %>
+      <% else %>
+        <%= t("metrics_summary.description.incomplete_week") %>
+      <% end %>
```

Comment:
could this not be a method in the View.rb  to return a summary description and then all you need to do is call it here. It would be easier to unit the hell out of it to make sure it returns what we expect it to or handle cases we dont expect.

Reply:
Done